samples/force-sensitive-resistor/Program.cs

```diff
@@ -5,54 +5,46 @@
 using System;
 using System.Threading;
 using Iot.Device.Adc;
+using force_sensitive_resistor;
 
-namespace force_sensitive_resistor
```

Comment:
I understand we are trying to showcase functional programming for c#9 here, but I wonder if we should stick with object oriented language given that these samples are supposed to be entry-level sort of examples?

Reply:
Also, I'm open for discussion here, if you think it's best to keep it as you have it I'm fine too.

Reply:
Seems like keeping that line would significantly reduce the value of top-level projects. Also seems like we're not losing much. I also don't think we're losing anything OO. It would be nice if you could declare the namespace with a single line that didn't require nesting the rest of the file.